lib/internal/Magento/Framework/Data/Form/Element/Collection.php

```diff
@@ -121,6 +126,7 @@ public function add(AbstractElement $element, $after = false)
                 if ($currElement->getId() == $after) {
                     $newOrderElements[] = $currElement;
                     $newOrderElements[] = $element;
+                    // phpcs:ignore Magento2.Performance.ForeachArrayMerge
```

Comment:
Should be possible to fix - https://github.com/magento/magento-coding-standard/issues/20

Reply:
Hello @xmav 
Thank you for your review.
It is not possible here.